src/osx/carbon/cursor.cpp

```diff
@@ -278,7 +278,26 @@ wxCursor::wxCursor(const wxString& cursor_file, wxBitmapType flags, int hotSpotX
     if ( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
     {
 #if wxOSX_USE_COCOA
-        wxFAIL_MSG( wxT("Not implemented") );
+        wxImage image;
+        wxFileName fileName( wxStandardPaths::Get().GetResourcesDir() + "/" + cursor_file + ".png" );
```

Comment:
This is really hopeless :-(

```suggestion
        wxFileName fileName( wxStandardPaths::Get().GetResourcesDir(), cursor_file, ".png" );
```

Reply:
Fixed